connectors/rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/RmqSourceReplicator.java

```diff
@@ -164,6 +166,11 @@ public boolean compare(Map<String, Set<TaskTopicInfo>> origin, Map<String, Set<T
         return true;
     }
 
+
+    @Override public void init(KeyValue config) {
+
```

Comment:
config should be init by init

Reply:
> config should be init by init

For the Replicator, the validate method is already config initialized. So it's not initialized in the init method.